docs/scos/dev/architecture/module-api/definition-of-module-api.md

```diff
@@ -22,48 +22,49 @@ redirect_from:
+## Public API
 
-* [Facades](/docs/scos/dev/back-end-development/zed/business-layer/facade/facade.html)
-* [Clients](/docs/scos/dev/back-end-development/client/client.html)
-* [Query Containers](/docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.html)
-* [Services](/docs/scos/dev/back-end-development/messages-and-errors/registering-a-new-service.html)
-* Plugin defining protected methods in dependency providers.
+In the Spryker Commerce OS’s core, the following is the public API:
 
-And the interfaces which are implemented everywhere are also part of the API:
+* Public methods in these locatable classes:
+    * [Facades](/docs/scos/dev/back-end-development/zed/business-layer/facade/facade.html)
+    * [Clients](/docs/scos/dev/back-end-development/client/client.html)
+    * [Query Containers](/docs/scos/dev/back-end-development/zed/persistence-layer/query-container/query-container.html)
+    * [Services](/docs/scos/dev/back-end-development/messages-and-errors/registering-a-new-service.html)
+    * Plugin defining protected methods in dependency providers.
 
-* Plugin interfaces
-* Plugins
-
-In addition to these obvious cases, there are some other classes that are part of the API and can cause a BC break:
-
-* module Config [`Client/Yves/Zed/Shared/Service`](/docs/scos/dev/back-end-development/data-manipulation/configuration-management.html)
-* Controllers
-* Twig functions
-* [CLI commands](/docs/scos/dev/back-end-development/console-commands/implementing-a-new-console-command.html)
-* Public constants that define environment configuration in [Constant Interfaces](/docs/scos/dev/back-end-development/data-manipulation/configuration-management.html)
-
-Every change in a schema can cause a BC break:
+* Interfaces:
+    * Plugin interfaces
+    * Plugins
 
+* Other classes:
+    * module Config [`Client/Yves/Zed/Shared/Service`](/docs/scos/dev/back-end-development/data-manipulation/configuration-management.html)
+    * Controllers
+    * Twig functions
+    * [CLI commands](/docs/scos/dev/back-end-development/console-commands/implementing-a-new-console-command.html)
+    * Public constants that define environment configuration in [Constant Interfaces](/docs/scos/dev/back-end-development/data-manipulation/configuration-management.html)
 * [Database](/docs/scos/dev/back-end-development/zed/persistence-layer/database-schema-definition.html)
 * Search
 * [Storage](/docs/scos/dev/back-end-development/client/using-and-configuring-redis-as-a-key-value-storage.html)
-* Changes in [transfer objects](/docs/scos/dev/back-end-development/data-manipulation/data-ingestion/structural-preparations/creating-using-and-extending-the-transfer-objects.html) can also cause BC breaks e.g. when an existing field is renamed.
+* [Transfer objects](/docs/scos/dev/back-end-development/data-manipulation/data-ingestion/structural-preparations/creating-using-and-extending-the-transfer-objects.html)
+* Embedded open-source components
```

Comment:
Shall we add an example or more detailed explanation? 
Or rename it to "dependencies to open-source components"

Reply:
We can add examples only if they are small enough to add as part of this bullet point. Otherwise, lets  add what you think is best